File: LLM-Translations/GPT-4/avatar/Python/C/codeforces_276_B.c
#include <stdio.h>
#include <stdbool.h>
#include <string.h>
#include <ctype.h>

int main() {
    char s[1000];
    scanf("%s", s);
    bool x = true;
    int dt[128] = {0};
    for (int i = 0; i < strlen(s); i++) {
        dt[s[i]]++;
    }
    int cnt = 0;
    x = true;
    for (int i = 0; i < 128; i++) {
        if (dt[i] % 2 != 0 && cnt < 1) {
            cnt++;
        } else if (dt[i] % 2 != 0 && cnt >= 1) {
            x = false;
            break;
        }
    }
    if (x) {
        printf("First\n");
    } else if (!x && strlen(s) % 2 == 0) {
        printf("Second\n");
    } else if (!x && strlen(s) % 2 != 0) {
        printf("First\n");
    }
    return 0;
}
// End of Code

File: LLM-Translations/GPT-4/avatar/Python/Go/codeforces_276_B.go
package main

import (
	"fmt"
)

func main() {
	var s string
	fmt.Scanln(&s)
	x := true
	dt := make(map[rune]int)
	for _, i := range s {
		if _, ok := dt[i]; ok {
			dt[i]++
		} else {
			dt[i] = 1
		}
	}
	cnt := 0
	x = true
	for _, value := range dt {
		if value%2 != 0 && cnt < 1 {
			cnt++
		} else if value%2 != 0 && cnt >= 1 {
			x = false
			break
		}
	}
	if x {
		fmt.Println("First")
	} else if !x && len(s)%2 == 0 {
		fmt.Println("Second")
	} else if !x && len(s)%2 != 0 {
		fmt.Println("First")
	}
}
// End of Code
